elixir/apps/web/lib/web/live/policies/new.ex

```diff
@@ -56,41 +59,51 @@ defmodule Web.Policies.New do
           <.form :if={@actor_groups != []} for={@form} phx-submit="submit" phx-change="validate">
             <.base_error form={@form} field={:base} />
             <div class="grid gap-4 mb-4 sm:grid-cols-1 sm:gap-6 sm:mb-6">
-              <.input
-                field={@form[:actor_group_id]}
-                label="Group"
-                type="group_select"
-                options={Web.Groups.Components.select_options(@actor_groups)}
-                value={@actor_group_id || @form[:actor_group_id].value}
-                disabled={not is_nil(@actor_group_id)}
-                required
-              />
-
-              <.input
-                field={@form[:resource_id]}
-                label="Resource"
-                type="select"
-                options={
-                  Enum.map(@resources, fn resource ->
-                    group_names = resource.gateway_groups |> Enum.map(& &1.name)
-
-                    [
-                      key: "#{resource.name} - #{Enum.join(group_names, ",")}",
-                      value: resource.id
-                    ]
-                  end)
-                }
-                value={@resource_id || @form[:resource_id].value}
-                disabled={not is_nil(@resource_id)}
-                required
-              />
-              <.input
-                field={@form[:description]}
-                type="textarea"
-                label="Description"
-                placeholder="Enter a reason for creating a policy here"
-                phx-debounce="300"
-              />
+              <fieldset class="flex flex-col gap-2">
+                <.input
+                  field={@form[:actor_group_id]}
+                  label="Group"
+                  type="group_select"
+                  options={Web.Groups.Components.select_options(@actor_groups)}
+                  value={@actor_group_id || @form[:actor_group_id].value}
+                  disabled={not is_nil(@actor_group_id)}
+                  required
+                />
+
+                <.input
+                  field={@form[:resource_id]}
+                  label="Resource"
+                  type="select"
+                  options={
+                    Enum.map(@resources, fn resource ->
+                      group_names = resource.gateway_groups |> Enum.map(& &1.name)
+
+                      [
+                        key: "#{resource.name} - #{Enum.join(group_names, ",")}",
+                        value: resource.id
+                      ]
+                    end)
+                  }
+                  value={@resource_id || @form[:resource_id].value}
+                  disabled={not is_nil(@resource_id)}
+                  required
+                />
+              </fieldset>
+
+              <.condition_form form={@form} providers={@providers} />
```

Comment:
Can we make this part of Team and higher, and give them a peek here at the form?

```suggestion
              <div class="flex justify-between">
                <div class={@policy_conditions_enabled && "" || "opacity-50"}>
                  <.condition_form form={@form} providers={@providers} />
                </div>
                <div :if={!@policy_conditions_enabled}>
                  UPGRADE TO UNLOCK
                </div>
              </div>
```

Reply:
<img width="757" alt="Screenshot 2024-05-03 at 12 43 24 PM" src="https://github.com/firezone/firezone/assets/167144/9e9277cb-4653-427c-ade3-4e3b9d479411">